site/content/docs/developing/sidecars.ja.md

```diff
@@ -1,16 +1,19 @@
 # サイドカー
 サイドカーは主となるコンテナと共に実行され補助的な役割を担うコンテナのことです。サイドカーの役割はロギングや設定ファイルの取得、リクエストのプロキシ処理などの周辺的なタスクを実行することです。
 
+!!! attention
+    Request-Driven Web Service はサイドカーの利用をサポートしていません。
+
+!!! Attention
+    メインコンテナに Windows イメージを使用している場合、[FireLens](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html), [AWS X-Ray](https://aws.amazon.com/xray/), [AWS App Mesh](https://aws.amazon.com/app-mesh/) はサポートされていません。お持ちのサイドカーコンテナで Windows をサポートしているか確認してください。
```

Comment:
- Replacing the links with the Japanese websites
- Adding more context to the last sentence (This can be optional I think)

```suggestion
    メインコンテナに Windows イメージを使用している場合、[FireLens](https://docs.aws.amazon.com/ja_jp/AmazonECS/latest/developerguide/using_firelens.html), [AWS X-Ray](https://aws.amazon.com/jp/xray/), [AWS App Mesh](https://aws.amazon.com/jp/app-mesh/) はサポートされていません。利用しようとしているサイドカーコンテナが Windows 環境での実行をサポートしているか確認してください。
```

Reply:
I love your localization! 👍